app/views/course/groups/_users.html.slim

```diff
@@ -0,0 +1,14 @@
+table.table
```

Comment:
_form.html.slim for the form. _user.html.slim for showing one user. This should belong in the parent view.